.php_cs

```diff
@@ -37,12 +38,14 @@ $config->fixers(
         'function_call_space',
         'function_declaration',
         'indentation',
+        'join_function',
```

Comment:
To be honest I don't get what's on your mind.
In composer there is latest stable version of PHP CS Fixer.
And what regression?